tasks/src/object-detection/data.ts

```diff
@@ -1,13 +1,17 @@
 import type { TaskDataCustom } from "../Types";
 
 const taskData: TaskDataCustom = {
-	datasets: [
-		{
-			// TODO write proper description
-			description: "Widely used benchmark dataset for multiple Vision tasks.",
-			id:          "merve/coco2017",
-		},
+		datasets: [
+	  {
+		description: "Widely used benchmark dataset for multiple Vision tasks.",
+		id: "merve/coco2017",
+	  },
+	  {
+		description: "Car Object Detection dataset with annotated bounding boxes around cars.",
```

Comment:
this one sounds good!

Reply:
Hi @merveenoyan, thank you so much for the review. Could you please clarify me please if I need to make any changes or what would be the next step I need to follow.

Reply:
I think there's only above review that you can edit and nothing else for now :)